.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -35,6 +35,8 @@ body:
           required: true
         - label: I am sure that this is a bug in LangChain rather than my code.
           required: true
+        - label: The bug occurs on the latest stable version of LangChain.
```

Comment:
Are folks going to have an easy time determining what this is? Given that we have a number of packages?